File: no-auth/src/main/java/com/kjj/noauth/NoAuthApplication.java
package com.kjj.noauth;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

@SpringBootApplication
@EnableDiscoveryClient
@ConfigurationPropertiesScan
public class NoAuthApplication {

    public static void main(String[] args) {
        SpringApplication.run(NoAuthApplication.class, args);
    }

}

File: no-auth/src/main/kotlin/com/kjj/noauth/NoAuthApplication.kt
package com.kjj.noauth

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@SpringBootApplication
@EnableDiscoveryClient
@ConfigurationPropertiesScan
class NoAuthApplication

fun main(args: Array<String>) {
    runApplication<NoAuthApplication>(*args)
}
